connector/src/snapshot.rs

```diff
@@ -9,12 +8,25 @@ use solana_client::{
 use solana_rpc::rpc::rpc_accounts::AccountsDataClient;
 use solana_sdk::{commitment_config::CommitmentConfig, slot_history::Slot};
 
-use crate::{AnyhowWrap, FilterConfig};
+use crate::AnyhowWrap;
+
+/// gPA snapshot struct
+pub struct SnapshotProgramAccounts {
+    pub slot: Slot,
+    pub accounts: Vec<RpcKeyedAccount>,
+}
+
+/// gMA snapshot struct
+pub struct SnapshotMultipleAccounts {
+    pub slot: Slot,
+    // (account pubkey, snapshot account)
+    pub accounts: Vec<(String, Option<UiAccount>)>,
+}
 
 pub async fn get_snapshot_gpa(
```

Comment:
I am not sure changing the signature of the problem will have issues other libraries depending on this code.